docs/resources/node_template.md

```diff
@@ -181,6 +181,19 @@ The following attributes are exported:
 * `tags` - (Optional) Comma-separated list of tags to apply to the Droplet (string)
 * `userdata` - (Optional) Path to file with cloud-init user-data (string)
 
+### `hetzner_config`
+
+#### Arguments
+
+* `api_token` - (Required) Hetzner Cloud project API token (string)
```

Comment:
This argument is defined as Required and Sensitive on schema, could you please specify it?? `(Required/sensitive)`